### DISCUSSION TITLE
Edit the default login view page

### DISCUSSION THREAD

**Discussion by Smailancer** [filamentphp/filament#11241] [Help] [answered]
### Package

Panel builder

### Package Version

v3

### How can we help you?

Hello There , So I'm new to to filament and I installed it after initiating my app using Jetstream to add AUTH system , now after adding Filament to my project I guess it uses the same Jetstream AUTH system but my problem here is I want to edit the size of the logo that appears in  the login page using tailwind classes but I cant find the right view file to edit , I run 

`php artisan vendor:publish --tag=filament-views`

 to publish the views but all what I get is the components while I'm looking for the AUTH page to edit , I don't know why I couldn't find anything about it in the documentation maybe there are some prior knowledge I should know before using filament

**Comment by devhoussam1998** [comment]
my friend follow this: https://filamentphp.com/docs/3.x/panels/users#customizing-the-authentication-features

**Reply by Smailancer** [reply]
Yeas I already saw this , what I want is just changing the size of the logo in the login page I don't want to create a whole new one to override the existent one

**Comment by leandrocfe** [comment, marked as answer]
You shouldn't publish views - https://filamentphp.com/docs/3.x/support/style-customization#publishing-blade-views

You can use this to achieve what you need
1 - [Create a custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme)
2 - Customize the css

```css
.fi-simple-header .fi-logo {
    @apply !h-20;
}
```
